Pflichtenheft.md

```diff
@@ -81,34 +106,50 @@ Anfrage des Kunden per Webseite ist optional
  - Preis
  - Status (geplant, beauftragt, in Durchführung, beendet, abgebrochen)
  - Anmerkungen (z.B. Abbruchgrund)
-- Produktleistungen
-- 
+- Mahnwesen
+ - Auftrag (Link zum Auftrag)
```

Comment:
Ich persönlich bevorzuge eine Desktopanwendung in C#. Mit VS haben wir hier ein richtig gutes Werkzeug bei der Hand.
Es geht hier aber nicht nach meiner Nase, sondern um einen Kompromiss (eventuell) von uns allen. In welcher Sprache programmiert ihr hauptsächlich? Vielleicht muss ja auch nur ich mich anpassen! Die größere Erfahrung in der Programmierung ist eindeutig bei euch zu finden.